docs/content/core/system-dns.md

```diff
@@ -24,15 +24,16 @@ always a fallback.
 
 ## Disabling the Resolver {#disable}
 
-If you run into issues with DNS resolution, try disabling the automatic resolver:
+If you run into issues with DNS resolution on the host machine, try disabling the build-in DNS resolver:
```

Comment:
I believe we mean to say `built-in DNS`, right @shelane ?

Reply:
Correct

Reply:
Fixed.